spotify/models/user.py

```diff
@@ -66,8 +125,14 @@ class User(URIBase):  # pylint: disable=too-many-instance-attributes
 
     _refresh_task: Optional[asyncio.Task]
 
-    def __init__(self, client: "spotify.Client", data: dict, **kwargs):
-        self.refresh_token = kwargs.pop("refresh_token", None)
+    def __init__(
```

Comment:
I believe in cheap creation and destruction, the changes to this `__init__` have made it much more considerably expensive to create `User` objects, this class is created in other parts of the library so any added cost here is added cost there.